src/fides/cli/commands/util.py

```diff
@@ -94,7 +94,7 @@ def worker(ctx: click.Context) -> None:
     Starts a celery worker.
     """
     # This has to be here to avoid a circular dependency
-    from fides.api.ops.tasks import start_worker
+    from fides.api.ops.worker import start_worker
```

Comment:
Thanks this is a nice semantic change

Reply:
👍 actually needed to split it out into its own module for dependency reasons, but i do think the organization now is a bit cleaner semantically.